lib/src/text_view.dart

```diff
@@ -164,14 +165,20 @@ class _RichTextViewState extends State<RichTextView> {
 
           final mapping = _mapping[matchText!] ??
               _mapping[_mapping.keys.firstWhere((element) {
-                final reg = RegExp(
+                var ret = false;
```

Comment:
Copied from our PatternTextEditingController to only match the pattern that fully captures the text. Issue was that regexp cought both link and bold patterns, then applied both patterns as it allowed for partial matching